types/evse_manager.yaml

```diff
@@ -200,7 +197,10 @@ types:
         description: Transaction finished meter value
         type: object
         $ref: /powermeter#/Powermeter
-      signed_meter_value:
+      start_signed_meter_value:
+        description: Signed meter value
+        type: string
+      stop_signed_meter_value:
         description: Signed meter value
```

Comment:
And say maybe on the interface level as well, why both meter values are at the "stop_transaction" event.